ckanext/oauth2/utils.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this file required? I don't see any references to it...

Reply:
Such file is linked imported by `ckanext/oauth2/cli.py`, but I think it is a mistake, and both the file and the import should be deleted.